[release/8.0.1xx-rc1] Updates for dotnet/runtime

rmarinho opened the pull request:
### Description of Change

- Remove CoherentParentDependency from Android on the `Microsoft.NETCore.App.Ref`
- Manual running the `darc update-dependencies --channel '.NET 8 RC 1' --debug` 
- merges android updates from here https://github.com/dotnet/maui/pull/16928

rmarinho committed:
Update dependencies for runtime

rmarinho committed:
Update versions

rmarinho committed:
Add feed for 7.0.11

rmarinho committed:
Update NuGet.config

rmarinho committed:
Update NuGet.config

jonathanpeppers reviewed (dismissed):
The idea here was exactly what I was thinking.

How hard would it be to do something like this with the internal/signed builds of .NET?

rmarinho commented:
> The idea here was exactly what I was thinking.
> 
> How hard would it be to do something like this with the internal/signed builds of .NET?

yeah that's what i was going to also try, get maui-release pipeline (that's the one that builds only packages to ship) is a very slim pipeline, i think i can modify it to run on dneng

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/xamarin/xamarin-android build 8.0.1xx-rc1-4e86e669ece90ae4411b27307d4c2d1619bc353f-1

Microsoft.Android.Sdk.Windows
 From Version 34.0.0-rc.1.420 -> To Version 34.0.0-rc.1.426

Dependency coherency updates

Microsoft.Dotnet.Sdk.Internal,Microsoft.NETCore.App.Ref,Microsoft.NET.Workload.Emscripten.Current.Manifest-8.0.100.Transport
 From Version 8.0.100-rc.1.23415.5 -> To Version 8.0.100-rc.1.23417.5 (parent: Microsoft.Android.Sdk.Windows

rmarinho committed:
Merge branch 'darc-release/8.0.1xx-rc1-07c89273-0781-40fe-b345-a41a1749c0b9' into coherent-build-rc1

# Conflicts:
#	eng/Version.Details.xml
